docs/userGuide/syntax/navBars.mbdf

```diff
@@ -2,11 +2,17 @@
 
 **Navbar allows visitors of your website to navigate through pages easily.**
 
+<box type="warning">
+  <markdown>
+Note: **Navbars** should be placed within a [header file]({{ baseUrl }}/userguide/tweakingthepagestructure.html#headers) to ensure correct positioning at the top of your page.
```

Comment:
Should we change "to ensure correct positioning" -> "to ensure that they are correctly positioned"?

Reply:
"... to ensure that they are correctly positioned above `page navigation` and `site navigation` menus." ?

Added more explicit interaction description.

Reply:
How about:

"... to ensure that they are correctly positioned at the top of the page, above the `page navigation` and `site navigation` menus."